core/controllers/feedback.py

```diff
@@ -32,6 +32,8 @@ def get(self, exploration_id):
         self.values.update({
             'threads': [t.to_dict() for t in feedback_services.get_all_threads(
                 exploration_id, False)]})
+        feedback_services.update_feedback_message_references(
```

Comment:
clear_feedback_message_references might be a better name? "Update" is a bit misleading...

Also, I feel like there might be weird coupling here -- it's possible that the editor page is opened and the feedback threads are loaded, but the user doesn't actually navigate to the feedback tab to see them. Maybe we should send a POST request if the user navigates to the feedback tab, to a separate handler, which is responsible for clearing the queue? That also preserves the general rule that GET handlers should not be state-changing.